dcicutils/misc_utils.py

```diff
@@ -1518,6 +1518,19 @@ def __repr__(self):
         return f"<{self.name}@{id(self):x}>"
 
 
+def key_value_pair(key, value):
+    return {'Key': key, 'Value': value}
+
+
+def merge_key_value_pairs(x, y):
+    merged = {}
```

Comment:
Again we might want to consider a name change to respond to @willronchetti's review comments. Maybe `merge_key_value_dict_lists` or something like that would be a better name? I didn't make such a change because I didn't have a good sense of what would be the right name, and I didn't want to hold up @alexander-veit's work. But I'm a bit flexible on this. The main thing is probably that it get a doc string, so...

```suggestion
def merge_key_value_pairs(x, y):
    """
    Merges two lists of the form [{"Key": k1, "Value": v1}, {"Key": k2, "Value": v2}].
    It is assumed that neither list has repeated keys, and that the resulting list also should not.
    Entries in the resulting list will be in the same order as the original two lists except when both
    lists contain entries that share a key. In that case, the two entries will be merged
    into a single entry with the position of the first such entry and value of the second.
    """
    merged = {}
```